### PR TITLE
User can create a todo.

### PR DESCRIPTION
**Pull request by ryancurrie**

### CODE REVIEW

**Comment by thebearingedge** [comment]
No `.gif`?

File: client/todo-form.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  handleSubmit(event) {
+    event.preventDefault()
+    const formData = new FormData(event.target)
+    const data = {
+      task: formData.get('task'),
+      dueDate: formData.get('dueDate')
+    }
+    console.log(JSON.stringify(data))
```
Comment (thebearingedge) [inline client/todo-form.js]: You can remove console logs like this.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Cool, just one tweak, and be sure to add your project config files (linter/sync.js/etc)

**Comment by ryancurrie** [comment]
todo-1

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please remove extra console logs.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**